.github/CONTRIBUTING.md

```diff
@@ -67,6 +67,37 @@ In addition to our automatically enforced [lint rules](https://github.com/bpmn-i
 
 __Rationale:__ People should be able to consume parts of the library with an ES module aware bundler such as [Webpack](https://webpack.js.org/) or [Rollup](https://rollupjs.org) without the need to use a transpiler such as [Babel](https://babeljs.io/).
 
+### Unit Tests
+
+In order to retrieve a sign-off for your contribution, it needs to be sufficiently and well tested. Please structure your unit tests into [**given**, **when** and **then**](https://gist.github.com/nikku/1dc2edb553565238c7bf). To increase overall readability and understandability please also leave two empty lines before each `describe(...)` and `it(...)` block unless it is the first statement in the current indentation level. Also see this example:
```

Comment:
> leave two empty lines before each `describe(...)` and `it(...)` block

Maybe "separate `it` and `describe` blocks on the same level with two empty lines"?

Reply:
But that would not include the two lines prior to a `it` or `describe` block after another `setup` block, would it?

Reply:
"separate `before`, `it` and `describe` blocks on the same level with two empty lines" would work for me just fine.

Generally speaking I'd be fine with some ambiguity here. If we really would like to enforce it, we could [create an eslint rule](https://github.com/bpmn-io/eslint-plugin-bpmn-io) that fixes things auto-magically.

Reply:
Fine to keep it as is, too.

What would be more important for me is that users see actual well written test cases linked => if I'm getting into a project I should contribute in the spirit of the project, even if no special style guide exists.

Reply:
Added examples :+1: